css/classPage.css

```diff
@@ -83,4 +83,13 @@ dialog[id^="photosDialog"]::backdrop {
 
 .tabela tr.k, .tabela td.k {
     background-image: none;
+}
+
+.tabela > tbody > tr, .tabela > tr {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+}
```

Comment:
This can be dangerous, in general we shouldn't mess with display in table elements